RELEASE.md

```diff
@@ -62,19 +63,21 @@ Following is the Release Compatibility Matrix for PyTorch releases:
 
 ## Release Cadence
 
-Following is the release cadence for year 2023/2024. All dates below are tentative, for latest updates on the release scheduled please follow [dev discuss](https://dev-discuss.pytorch.org/c/release-announcements/27). Please note: Patch Releases are optional.
+Following is the release cadence. All future dates below are tentative, for latest updates on the release scheduled please follow [dev discuss](https://dev-discuss.pytorch.org/c/release-announcements/27). Please note: Patch Releases are optional.
 
 | Minor Version | Release branch cut | Release date | First patch release date | Second patch release date|
 | --- | --- | --- | --- | --- |
 | 2.1 | Aug 2023 | Oct 2023 | Nov 2023 | Dec 2023 |
 | 2.2 | Dec 2023 | Jan 2024 | Feb 2024 | Mar 2024 |
 | 2.3 | Mar 2024 | Apr 2024 | Jun 2024 | Not planned |
-| 2.4 | Jun 2024 | Jul 2024 | (Sept 2024) | Not planned |
-| 2.5 | Sep 2024 | Oct 2024 | (Nov 2024) | (Dec 2024) |
-| 2.6 | Dec 2024 | Jan 2025 | (Feb 2025) | (Mar 2025) |
+| 2.4 | Jun 2024 | Jul 2024 | Sept 2024 | Not planned |
+| 2.5 | Sep 2024 | Oct 2024 | Nov 2024 |  Not planned |
+| 2.6 | Dec 2024 | Jan 2025 | Not planned | Not planned |
 | 2.7 | Mar 2025 | Apr 2025 | (May 2025) | (Jun 2025) |
```

Comment:
are we still doing second patch releases?  Isn't that only done for emergency situations now?

Reply:
Yes these act as placeholders for possible patch release dates. For patch release emergency situations is probably not correct term. Patch releases are done when there are issues that are critical enough to perform patch release.